tensorflow_decision_forests/contrib/temporal_feature_processor/temporal_feature_processor/interval.py

```diff
@@ -0,0 +1,3 @@
+from pandas import Timedelta as Interval
```

Comment:
Shouldn't this be in `implementation/pandas`?

Reply:
yep! good catch